### ISSUE TITLE
Extending resolvers in `createSchema` not working

### ISSUE THREAD

**Issue by fjcero** [nestjs/graphql#14]
This syntax is not working in the NPM version. Probably a release is pending.

```js
const resolvers = {
  UUID: GraphQLUUID
}

const typeDefs = this.graphQLFactory.mergeTypesByPaths('./**/*.graphql');
const schema = this.graphQLFactory.createSchema({ typeDefs, resolvers });
```

**Comment by yuluyi** [comment]
+1
Resolvers from this.resolversExplorerService.explore() will overwrite the ones in schemaDefinition

```ts
createSchema(schemaDefintion = { typeDefs: {} }) {
        return graphql_tools_1.makeExecutableSchema(Object.assign({}, schemaDefintion, { resolvers: this.resolversExplorerService.explore() }));
}
```

**Comment by kamilmysliwiec** [comment]
This issue is fixed in the latest version - `v3.0.0`.
